core/jazz_ui/src/app/pages/services-list/services-list.component.scss

```diff
@@ -8,6 +8,20 @@
 /deep/ .table-wrap.fixed {
   position: fixed;
 }
+/deep/.service-list-table
+ .column:nth-child(3) 
+ .input-wrapper {
+    margin-left: 0 !important;
+}
+/deep/ .service-list-table 
```

Comment:
Avoid use of deep where currently possible.

Reply:
ok